net.worldpainter.worldpainter.metainfo.xml

```diff
@@ -0,0 +1,146 @@
+    <release version="v2.5.7" date="2018-09-05" type="stable"/>
+    <release version="v2.5.6" date="2018-07-28" type="stable"/>
+    <release version="v2.5.5" date="2018-07-04" type="stable"/>
+    <release version="v2.5.4" date="2018-06-22" type="stable"/>
+    <release version="v2.5.3" date="2018-06-18" type="stable"/>
+    <release version="v2.5.2" date="2018-06-16" type="stable"/>
+    <release version="v2.5.1" date="2018-05-08" type="stable"/>
+    <release version="v2.5.0" date="2018-04-23" type="stable"/>
+    <release version="v2.4.8" date="2018-04-21" type="stable"/>
+    <release version="v2.4.7" date="2018-04-18" type="stable"/>
+    <release version="v2.4.6" date="2018-04-18" type="stable"/>
+    <release version="v2.4.5" date="2018-04-17" type="stable"/>
+    <release version="v2.4.4" date="2018-04-13" type="stable"/>
+    <release version="v2.4.3" date="2018-02-05" type="stable"/>
+    <release version="v2.4.2" date="2018-02-01" type="stable"/>
+    <release version="v2.4.1" date="2017-02-05" type="stable"/>
+    <release version="v2.4.0" date="2017-02-04" type="stable"/>
+    <release version="v2.3.6" date="2017-01-05" type="stable"/>
+    <release version="v2.3.5" date="2017-01-04" type="stable"/>
+    <release version="v2.3.4" date="2016-12-26" type="stable"/>
+    <release version="v2.3.3" date="2016-12-23" type="stable"/>
+    <release version="v2.3.2" date="2016-12-09" type="stable"/>
+    <release version="v2.3.1" date="2016-12-04" type="stable"/>
+    <release version="v2.3.0" date="2016-11-27" type="stable"/>
+    <release version="v2.2.6" date="2016-10-25" type="stable"/>
+    <release version="v2.2.5" date="2016-10-23" type="stable"/>
+    <release version="v2.2.4" date="2016-08-20" type="stable"/>
+    <release version="v2.2.3" date="2016-08-07" type="stable"/>
+    <release version="v2.2.2" date="2016-08-06" type="stable"/>
+    <release version="v2.2.1" date="2016-08-06" type="stable"/>
+    <release version="v2.2.0" date="2016-03-18" type="stable"/>
+    <release version="v2.1.6" date="2016-02-07" type="stable"/>
+    <release version="v2.1.5" date="2016-02-07" type="stable"/>
+    <release version="v2.1.4" date="2016-01-17" type="stable"/>
+    <release version="v2.1.3" date="2015-11-29" type="stable"/>
+    <release version="v2.1.2" date="2015-11-24" type="stable"/>
+    <release version="v2.1.1" date="2015-11-23" type="stable"/>
+    <release version="v2.1.0" date="2015-11-22" type="stable"/>
+    <release version="v2.0.2" date="2015-09-27" type="stable"/>
+    <release version="v2.0.1" date="2015-09-18" type="stable"/>
+    <release version="v2.0.0" date="2015-09-13" type="stable"/>
+    <release version="v1.11.2" date="2015-06-28" type="stable"/>
+    <release version="V1.11.1" date="2015-04-26" type="stable"/>
+    <release version="V1.11.0" date="2015-04-25" type="stable"/>
+    <release version="v1.10.6" date="2015-03-07" type="stable"/>
+    <release version="v1.10.5" date="2015-03-01" type="stable"/>
```

Comment:
How important is this list of releases? Does the current release need to be in it? It would be good if this whole process could be fully automated using Maven and require no manual actions such as editing files upon each release.